src/app/public/modules/inline-form/types/inline-form-button.ts

```diff
@@ -0,0 +1,9 @@
+import {
+  SkyInlineFormButtonAction
+} from './inline-form-button-action';
+
+export interface SkyInlineFormButton {
```

Comment:
This interface is exactly the same as the `SkyInlineFormButtonConfig` interface?